libr/bin/format/mach0/mach0.c

```diff
@@ -3135,9 +3144,9 @@ static void _parse_function_start_symbols(RBinFile *bf, struct MACH0_(obj_t) *mo
 		while (temp + 3 < temp_end && *temp) {
 			temp = r_uleb128_decode (temp, NULL, &value);
 			address += value;
-		//	RBinSymbol *sym = RVecRBinSymbol_push_back (&mo->symbols_vec);
-			RBinSymbol *sym = r_vector_end (&mo->symbols_cache);
-			// RBinSymbol *sym = r_vec_emplace_back (&mo->symbols_vec);
+			RBinSymbol *sym = RVecRBinSymbol_emplace_back (mo->symbols_vec);
+			// probably not necessary if we fill all the fields below, just in case.. but maybe we can have another rvec method for this
```

Comment:
emplace_back0 that calls emplace_back and memset for you? something to think about.

Reply:
sounds good to me!